src/Symfony/Component/Translation/Extractor/PhpExtractor.php

```diff
@@ -295,6 +329,16 @@ protected function parseTokens(array $tokens, MessageCatalogue $catalog, string
                     $normalizedFilename = preg_replace('{[\\\\/]+}', '/', $filename);
                     $metadata['sources'][] = $normalizedFilename.':'.$tokens[$key][2];
                     $catalog->setMetadata($message, $metadata, $domain);
+
+                    if (!empty($variables)) {
+                        $catalog->setMetadata($message, ['notes' => [
+                            [
+                                'category' => 'symfony-extractor-variables',
```

Comment:
Any preference for the category attribute value?

Reply:
No, but it should maybe use a constant somewhere, right?

Reply:
Here is my proposal: https://github.com/symfony/symfony/pull/38884/commits/46d26ec05131dd2d1ddaa76efb5f69be31786a1e (was not easy to choose a good place for the constants, I hope it's fine)

I moved it to a dedicated commit to be able to check/revert it easily.